JDI.Light/JDI.Light.Tests/Tests/Common/FileInputTests.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using JDI.Light.Tests.UIObjects;
+using NUnit.Framework;
+using OpenQA.Selenium;
```

Comment:
a lot of using statements are not required:
![image](https://user-images.githubusercontent.com/6689884/54680497-10d36380-4b1b-11e9-81a9-992f1c03caf3.png)